anitya/ui.py

```diff
@@ -10,8 +11,26 @@
 import anitya.lib.exceptions
 import anitya.lib.model
 
-from anitya.app import APP, SESSION, login_required
 from anitya.lib import utilities
+from anitya.lib.model import Session as SESSION
+
+
+ui_blueprint = flask.Blueprint(
+    'anitya_ui', __name__, static_folder='static', template_folder='templates')
+
+
+def login_required(function):
+    ''' Flask decorator to retrict access to logged-in users. '''
```

Comment:
Normally, I'd be all for this (you know how much I like documentation), but I also plan on replacing this with [flask-login's version](https://flask-login.readthedocs.io/en/latest/#flask_login.login_required) because it integrates nicely with [python-social-auth](http://python-social-auth.readthedocs.io/en/latest/configuration/flask.html)